Bugfix/multiple bug fixes

hannyle opened the pull request:
### Description

<!-- Please include a summary of the change or any information deemed important. -->

### Related issues
<!-- Reference any follow up issues with "Fixes #<issue-num>." -->

### Type of change

<!-- Replace [ ] with [x] to select options. -->
<!-- Please delete options that are not relevant. -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (this needs a follow up PR)

### Changes Made

<!-- List changes made. -->

### Testing

<!-- Are any tests part of this PR. -->
<!-- Replace [ ] with [x] to select options. -->
<!-- Please delete options that are not relevant. -->

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Tests do not apply
- [ ] Needs testing (start an issue or do a follow up PR about it)

### Mentions
<!-- Shout outs to your friends that you made this happen or need help. -->

sampsapenna commented on swift_browser_ui_frontend/src/common/lang.js:
```diff
@@ -615,6 +625,14 @@ let default_translations = {
         share: "Jaa",
         share_id: "Jaa tunnus",
+        share_id_tooltip:
+          "When another project wants share a folder " +
```
Initial translation for this:
Mikäli toisesta projektista halutaan jakaa kansio projektisi kanssa: Valitse tämä nappi ja lähetä Jakotunnus toisen projektin jäsenelle. Halutessasi jakaa kansion toisen projektin kanssa, pyydä jakotunnusta joltakin kyseisen projektin jäseneltä.

hannyle replied:
I added the translation. Thanks 👍

sampsapenna reviewed (changes requested):
Added a comment about translation, it can be substituted into the Finnish tooltip if need be

blankdots commented:
in order for this to pass in python unit tests, you need #893

sampsapenna reviewed (changes requested):
There are still linter errors present in JS code, so run `npm run lint` before merging. Otherwise LGTM, so I'll directly approve after that 👍🏼

hannyle commented:
> There are still linter errors present in JS code, so run `npm run lint` before merging. Otherwise LGTM, so I'll directly approve after that 👍🏼

Thank you for pointing that out. I fixed the Lint issue and rebased on `devel` branch
